src/drive/web/modules/queries.js

```diff
@@ -325,6 +325,38 @@ export const buildEncryptionByIdQuery = id => ({
   }
 })
 
+/**
+ * Provide selector and options to fetch files
+ *
+ * @param {string} bookmark - The query bookmark
+ * @returns {{options: {MangoQueryOptions}, selector: object}} A minimalist file list
+ */
+export const prepareSuggestionQuery = (bookmark = '') => {
+  const selector = {
+    _id: {
+      $gt: null
+    }
+  }
+  const options = {
+    partialFilter: {
+      _id: {
+        $ne: TRASH_DIR_ID
+      },
+      path: {
+        $or: [{ $exists: false }, { $regex: '^(?!/.cozy_trash)' }]
```

Comment:
nit: as this predicate on the `path` shouldn't be necessary if the trash was more reliable, it would be worth addding a comment here to mention it